site/content/Getting Started/_index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Getting Started"
+linkTitle: "Getting Started"
+weight: 2
+description: >
+    Taking VHS for a spin.
+---
+
+The quickest way to see `vhs` in action is to use the development Docker image in this repo to run a demo.
+
+## Prerequisites
+The first step is to clone the VHS repository.
+
+```
+git clone https://github.com/rename-this/vhs.git
+```
+
+You will need a working Docker installation to successfully execute the following commands. You should be able to
+install Docker from your favorite package manager, or you can see the [Docker site](https://www.docker.com/get-started)
+for more details.
+
+Once you have Docker set up and working, you can build the development container (which installs some helpful tools
+and mounts the source in the container) by changing directory into the repository you cloned and running:
+
+```
```

Comment:
Might be worth adding a note or two about what `make dev` is doing.

Reply:
Added a description of the make dev process and the development container.